### PR TITLE
Create C03_03

### PR DESCRIPTION
**Pull request by DeedeeDL**
The program will calculate the sum of n numbers (in this example, n is 7, so it calculates the sum of first 7 numbers)

### CODE REVIEW

File: src/main/java/org/example/C03_03_SumOfNNumbers.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.example;
+
+public class C03_03_SumOfNNumbers {
+    public static void main(String[] args) {
+        int n = 7;
+        int suma = 0;
```
Comment (andreiursudev) [inline src/main/java/org/example/C03_03_SumOfNNumbers.java]: Use english.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/C03_03_SumOfNNumbers.java (hunk 1 above)
Comment (DeedeeDL) [reply src/main/java/org/example/C03_03_SumOfNNumbers.java]: Done

File: src/main/java/org/example/C03_03_SumOfNNumbers.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    public static void main(String[] args) {
+        int n = 7;
+        int sum = 0;
+
+
+        for (int i = 0; i <= n; i++) {
+            sum = sum + i;
+
+        }
+        System.out.println("Suma numerelor este de: " + sum);
```
Comment (andreiursudev) [inline src/main/java/org/example/C03_03_SumOfNNumbers.java]: Use english.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/C03_03_SumOfNNumbers.java (hunk 2 above)
Comment (DeedeeDL) [reply src/main/java/org/example/C03_03_SumOfNNumbers.java]: done

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**